Darktrace/threat-visualizer/tests/test_anomalous_file.json

```diff
@@ -0,0 +1,1093 @@
+    "event": {
+      "category": "network",
+      "end": "2023-06-28T11:53:50Z",
+      "kind": "alert",
+      "type": ["info"]
+    },
+    "@timestamp": "2023-06-28T15:51:42Z",
+    "device": {
+      "id": "16"
+    },
+    "observer": {
+      "name": "Darktrace",
+      "product": "Threat visualizer"
+    },
+    "darktrace": {
+      "threat_visualizer": {
+        "commentCount": 0,
+        "pbid": 26316,
+        "time": 1687967502000,
+        "creationTime": 1687967508000,
+        "model": {
+          "then": {
+            "name": "AnomalousFile::ZiporGzipfromRareExternalLocation",
+            "pid": 619,
+            "phid": 9945,
+            "uuid": "80010119-6d7f-0000-0305-5e0000000172",
+            "logic": {
+              "data": [19046],
+              "type": "componentList",
+              "version": 1
+            },
+            "throttle": 3600,
+            "sharedEndpoints": false,
+            "actions": {
+              "alert": true,
+              "antigena": {},
+              "breach": true,
+              "model": true,
+              "setPriority": false,
+              "setTag": false,
+              "setType": false
+            },
+            "tags": ["", "AP:Tooling", "OTEngineer"],
+            "interval": 0,
+            "delay": 0,
+            "sequenced": false,
+            "active": true,
+            "modified": "2023-06-2811:53:50",
+            "activeTimes": {
+              "devices": {},
+              "tags": {},
+              "type": "exclusions",
+              "version": 2
+            },
+            "autoUpdatable": true,
+            "autoUpdate": true,
+            "autoSuppress": true,
+            "description": "AdevicehasdownloadedaZIPfilefromalocationthatthenetworkdoesnotnormallyvisit.\n\nAction:Reviewthefile,itshashandthesourcetoensurethatthisfileisrequiredwithinthenetworkforbusinesspurposes.",
+            "behaviour": "decreasing",
+            "created": {
+              "by": "System"
+            },
+            "edited": {
+              "by": "System"
+            },
```

Comment:
```suggestion
```

Reply:
I'm not sure of what you meant @squioc . Do you want them to be simple variables like created_by and edited_by ?